### ISSUE TITLE
run detection on multiple images at once 

### ISSUE THREAD

**Issue by shwingbu** [theAIGuysCode/yolov4-custom-functions#40]
Hi, I very much appreciate the work you've done here.
I was wondering if there's a way to edit the code so that I can run detection on multiple images and save results in a text file. The results being the coordinates and the image name. (Trying to create more training data for a larger dataset)

**Comment by Shubham1112** [comment]
Yes, please if you found the answer to your question. Can you please share it here?

**Comment by Youssef1406** [comment]
if i am not mistaken you could, run the code of process the same one you used to train darknet, that would change them into multiple of txt file that have all directory of each image and you could run your code on them, but i am not sure
